yogstation/code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -2,4 +2,10 @@
 	name = "medical supply radio encryption key"
 	desc = "An encryption key for mining medic headsets. To access the medical channel, use :m. For cargo, use :u."
 	icon_state = "cargo_cypherkey"
-	channels = list("Supply" = 1, "Medical" = 1)
\ No newline at end of file
+	channels = list("Supply" = 1, "Medical" = 1)
+
+/obj/item/encryptionkey/headset_medsec
+	name = "medical supply radio encryption key"
```

Comment:
nor this

Reply:
copied from the mining medic's headset, forgot to change it